### PR TITLE
Add Streak tasks which are generated from notes

### LINKED ISSUE #88: Have really robust system for having staff do tasks from check-ins
Expected behavior: whenever a CheckIn is created, create an associated task on the leader's Streak box and assign it to Kyle to follow-up with them.

Task content: "Follow-up on notes from check-in: <insert notes>"

### PR DESCRIPTION
**Pull request by paked**
This PR closes #88.

Things to test:

- Tasks get assigned to the right person
- Tasks are generated when there are notes outputted from a conversation. (1. if there wasn't a meeting. 2. if there was a meeting)

I've already configured our production environment to put @kyleemile as the default assignee.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ SLACK_TEAM_ID
 
 # Secret code required when redeeming .tech domains
 TECH_DOMAIN_REDEMPTION_SECRET_CODE
+
+# Who to assign Streak tasks to
+DEFAULT_TASK_ASSIGNEE
```
Comment (zachlatta) [inline README.md]: What if this was called `DEFAULT_STREAK_TASK_ASSIGNEE_EMAIL`? Or just `DEFAULT_STREAK_TASK_ASSIGNEE`?

File: app/jobs/leader_check_ins_job.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@ def perform(streak_keys = [])
 
   private
 
-  def retrieive_user_ids(streak_keys)
+  def retrieve_user_ids(streak_keys)
```
Comment (zachlatta) [inline app/jobs/leader_check_ins_job.rb]: Was this change intentional? Seems separate from the rest of the changes in this PR.

File: config/secrets.yml [hunk 3]
```diff
@@ -32,6 +32,7 @@ common: &common
   tech_domain_redemption_secret_code: <%= ENV.fetch("TECH_DOMAIN_REDEMPTION_SECRET_CODE") %>
 
   giphy_api_key: <%= ENV.fetch("GIPHY_API_KEY") %>
+  default_task_assignee_email: <%= ENV.fetch("DEFAULT_TASK_ASSIGNEE_EMAIL") %>
```
Comment (zachlatta) [inline config/secrets.yml]: This is different from what you wrote in the README, FYI.

File: app/jobs/leader_check_ins_job.rb (hunk 2 above)
Comment (MaxWofford) [inline app/jobs/leader_check_ins_job.rb]: @paked This will already be covered in https://github.com/hackclub/api/pull/116/files#diff-901c18c30d3bc30f13b107f181a0f6fdL31.
